### PR TITLE
KEYCLOAK-1918 - Add description field to client definition.

### PR DESCRIPTION
**Pull request by thomasdarimont**
Introduced a new description field with support for i18n for more descriptive client information.
Applications can use the description to display a "slightly" longer gist of
what the client / application is about, especially useful for tooltips.

The description is currently limited to 255 characters.

### CODE REVIEW

File: model/jpa/src/main/java/org/keycloak/models/jpa/entities/ClientEntity.java [hunk 1]
```diff
@@ -34,6 +34,8 @@
     private String id;
     @Column(name = "NAME")
     private String name;
+    @Column(name = "DESCRIPTION", length = 255)
```
Comment (stianst) [inline model/jpa/src/main/java/org/keycloak/models/jpa/entities/ClientEntity.java]: No need to use length here as we don't use Hibernate to generate the schema

**Comment by stianst** [comment]
Thanks, one last minor thing is the comment I added to ClientEntity

**Comment by thomasdarimont** [comment]
Removed the length definition, anything else?

**Comment by stianst** [comment]
Nopes, that's it I'm finished bitching ;)

Thanks for the PR I'll merge it once Travis finishes

**Comment by thomasdarimont** [comment]
Cool - thanks for the quick turn-around :)

**Pull request merged**